src/blocks/click-to-tweet/deprecated.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable sort-keys */
```

Comment:
why do you disable the sort-keys and then sort them anyway?

Reply:
> why do you disable the sort-keys and then sort them anyway?

This gave me a good laugh! Thank you for calling me out here. I think what I did was sort a few of them realize that the file also wanted me to sort all the old deprecations.